src/components/calcite-flow-item/calcite-flow-item.tsx

```diff
@@ -368,12 +372,20 @@ export class CalciteFlowItem {
   renderFab(): VNode {
     const hasFab = getSlotted(this.el, SLOTS.fab);
     return hasFab ? (
-      <div class={CSS.fabContainer} slot={PANEL_SLOTS.fab}>
+      <div key="fab-container" class={CSS.fabContainer} slot={PANEL_SLOTS.fab}>
         <slot name={SLOTS.fab} />
       </div>
     ) : null;
   }
 
+  renderDefaultSlot(): VNode {
+    return (
+      <div key="default-slot-container" class={CSS.defaultSlotContainer}>
```

Comment:
Just curious if `key` is needed here since this node is not rendered conditionally.